### DISCUSSION TITLE
GitHub has blocked Ukrainian IP addresses for Copilot

### DISCUSSION THREAD

**Discussion by insky** [community/community#47640] [Copilot Conversations]
### Select Topic Area

General

### Body

Please return my money

**Comment by tuves** [comment]
Thank you for reporting this. The bug fix has now been deployed to production and affected users should be able to connect. 

Due to the recency of this incident, we are still investigating the contributing factors and will provide a more detailed update on cause and remediation in the February Availability Report, which we will publish the first Wednesday of March.

We sincerely apologize for any stress and/or frustration this has caused.
